scripts/libs.js

```diff
@@ -15,6 +15,51 @@ export const globalConfig = {
   },
 };
 
+export const metaTags = {
+  componentsConfig: {
+    metaName: 'components-config',
+    fallbackContent: 'components-config.json',
+    // contentType: 'path without extension',
+  },
+  header: {
+    metaName: 'header',
+    fallbackContent: 'header.plain.html',
```

Comment:
I am thinking about starting a convention the have all the content we are loading as some sort of fragment in a folder call `fragments` or `global`, wdyt?

Reply:
I was thinking about the same for content loaded as fragments to be in a root folder `en/fragments`. 
One of the advantages is that we can then set a meta `robots noindex, nofollow` generally to that path.

But should we enforce it code wise? Now in my drafts folder I have a 'fragments' folder but If we enforce the above code wise we would not be able to load content from another place.

We can discuss in the spec meeting today.